### DISCUSSION TITLE
ZXing lib crashes in debug build

### DISCUSSION THREAD

**Discussion by darkhelmet72** [zxing-cpp/zxing-cpp#562] [Q&A]
1. Build ZXing for Windows 10
2. App write and build in VS 2022 
3. Built a library in static layout mode for release and debug mode app. In release mode, the library performs perfectly. in debug mode, the library crashed when the method - "ReadBarcode" is called.
It is worth noting that it falls only in high-resolution photos.

Can you tell me if this is a problem in the library or if some specifics are required when building the ZXing library for debug mode?

**Comment by axxel** [comment]
The problem is likely an assert. Please provide the details of the crash and also the input image that is causing the crash.
